parser/APIWriter.py

```
@@ -14,25 +16,30 @@ class APIWriter(Writer):
+                "Content-Type": "application/json",
+                API_KEY: API_KEY_VALUE,
+            }
             payload = {
                 "table": table,
                 "data": data,
             }
             # Calculate the size of the payload
             payload_size = len(json.dumps(payload))
             print("Payload Size:", payload_size)  # Print the payload size
-            response = requests.post(url=API_ENDPOINT, headers=headers, json=payload)
+            response = requests.post(
```

Ruff (0.13.1)

Probable use of `requests` call without timeout

(S113)